Makefile

```diff
@@ -201,3 +203,13 @@ local-a4:
 		$(VIRTUAL_ENV)/bin/python manage.py migrate; \
 		npm link ../adhocracy4; \
 	fi
+
+.PHONY: celery-worker-start
+celery-worker-start:
+	$(VIRTUAL_ENV)/bin/celery --app adhocracy-plus worker --loglevel INFO
+
+
+.PHONY: celery-worker-status
+celery-worker-status:
+	$(VIRTUAL_ENV)/bin/celery --app adhocracy-plus inspect registered
+	$(VIRTUAL_ENV)/bin/celery --app adhocracy-plus call dummy_task | awk '{print "celery-task-meta-"$$0}' | xargs redis-cli get | python3 -m json.tool
```

Comment:
should this go into a separate command? I find it a bit weird/unexpected behavior that asking for the status also executes a task

Reply:
done